src/install_start.sh

```diff
@@ -12,24 +12,16 @@ apt-get -y upgrade
 apt-get install -y dialog apt-utils
 
 # ******************************************************************************
-# 2. Setup to ignore noveau drivers so nvidia drivers run
```

Comment:
Removing this seemed to not affect anything. Is there somewhere you expect this to break things?

I'm generally a fan of removing as much as possible.

Reply:
To be clear this installs only the drivers, but the but you updated `install_nvidia_docker.sh` to nvidia-docker version 2.  So the tensorflow docker image handles installing nvidia-docker correct?

Also you use the ubuntu ppa for nvidia drivers.  This seems cool given the ppa updates

Reply:
Did you mean, does the Tensorflow Docker image install Cuda? Answer is yes.

We only need nvidia-docker-v2 (v1 not needed).